ideascube/models.py

```diff
@@ -311,6 +311,13 @@ def get_prep_value(self, value):
         return json.dumps(value)
 
 
+class LanguageField(models.CharField):
+    def __init__(self, *args, **kwargs):
+        kwargs['choices'] = get_all_languages()
+
```

Comment:
> Yet the former is much less readable.

That is *subjective*, and it's my purpose here. ;)
pep8 is about setting up conventions so we reduce the subjective aspects of code formatting, and thus the energy needed to read someone else code.
And this is why I think we should restrict as much as possible personal formatting.
So I think we should follow a mantra like: "if a space or a blank line is not required by pep8 we should not add it".
That said, I'm not arguing about this specific line, but more about our general habits as a team.